### ISSUE TITLE
VIC Proxy for registry

### ISSUE THREAD

**Issue by NixGi** [vmware/vic#8636]
Hi,
i have a Question About configuring a Proxy for the vic appliance registry. I try to verify the global dockerhub registry. But get the following error message:
Error connecting to https://registry.hub.docker.com: Connection timeout
I am behind a Proxy Server and added the Proxy Settings during the initial vic deployment. 
Also i try to Change the Admiral custom config file to add the Proxy Servers.
After restarting the Service, the Change is made in the config.properties file. but also the Default lines 
egistry.proxy=__null
registry.no.proxy.list=__null
 are still visible.

Is this the wrong place to configure the Proxy for the docker registry?
I have installed the latest Version 1.5.5.
Any hint will be appreciated
Frank

**Comment by wvermeulen** [comment]
Hi, I have the same issue. Have you found a why to get this to work?

**Comment by NixGi** [comment]
no. I have ask my colleagues to get Internet Access without adding a Proxy.
Sorry
